### ISSUE TITLE
Test failures for TLS

### ISSUE THREAD

**Issue by cmeiklejohn** [lasp-lang/partisan#59]
cc: @seancribbs 

```
2017-06-04 22:08:10.417
Ending test case default_manager_test

%%% partisan_SUITE ==> default_manager_test (group with_tls): FAILED
%%% partisan_SUITE ==> {{assertEqual,[{module,partisan_SUITE},
               {line,888},
               {expression,"ExpectedRand"},
               {expected,0.4626377977231828},
               {value,0.7612390866620362}]},
```

**Comment by lrascao** [comment]
is this happening mostly on Travis or locally?

**Comment by cmeiklejohn** [comment]
This is happening on my Windows machine.  Any thoughts @seancribbs?

**Comment by seancribbs** [comment]
There's nothing obvious about why that would fail on TLS vs. regular.

**Comment by lrascao** [comment]
maybe it's a windows thing, does `priv/lager/*` show any clue?

**Comment by seancribbs** [comment]
For what it's worth, if `make_certs.erl` doesn't work on Windows, we will be out of luck. It requires a working `openssl` CLI tool.

**Comment by cmeiklejohn** [comment]
This running inside of Ubuntu on Windows; so, there is a `openssl` command line available.  Is this just a timing issue?

**Comment by seancribbs** [comment]
That sounds likely. Do you have a `wait_until` function we could use?

**Comment by cmeiklejohn** [comment]
Resolved by #62.
